network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/tck/MergeNetworkTest.java

```diff
@@ -39,32 +40,117 @@ public void multipleDanglingLinesInMergedNetwork() {
     }
 
     @Override
-    public void testMergeSameId() {
+    public void test() {
         // FIXME
     }
 
     @Override
-    public void test() {
+    public void mergeThenCloneVariantBug() {
         // FIXME
     }
 
     @Override
-    public void xnodeNonCompatible() {
+    public void multipleDanglingLinesInMergingNetwork() {
         // FIXME
     }
 
     @Override
-    public void mergeThenCloneVariantBug() {
+    public void checkMergingSameFormat() {
         // FIXME
     }
 
     @Override
-    public void multipleDanglingLinesInMergingNetwork() {
+    public void testMergeAndDetach() {
         // FIXME
     }
 
     @Override
-    public void checkMergingSameFormat() {
+    public void testMergeAndDetachWithExtensions() {
+        // FIXME
+    }
+
+    @Override
+    public void failDetachWithALineBetween2Subnetworks() {
+        //FIXME
+    }
+
+    @Override
+    public void failDetachIfMultiVariants() {
+        //FIXME
+    }
+
+    @Test
```

Comment:
we can use @Override to be consistent with other files except for failMergeOnlyOneNetwork() failMergeOnSubnetworks() failMergeSubnetworks() failMergeContainingSubnetworks() testNoEmptyAdditionalSubnetworkIsCreated() that are package private in the abstract parent class

Reply:
sure, done